pom.xml

```diff
@@ -193,4 +187,25 @@
         </pluginManagement>
     </build>
 
+    <profiles>
+        <profile>
+            <id>all</id>
```

Comment:
why would we need this `all` profile?

Reply:
That's default - to build all modules if no profile provided.

Let's confirm scope:
- we need to get rid of skipUiBuild and replace it with profile
- add frontend as part of local build

Is that all? 

Then I suggest declare frontend as separate module - that allows us to avoid using properties to skip build stages